<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FanProfileDialog</class>
 <widget class="QDialog" name="FanProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GWEpp: New fan profile</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3" colspan="3">
    <widget class="QLineEdit" name="lineEdit_profile_name"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_profile_name">
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="3">
    <widget class="QSlider" name="horizontalSlider_fan_speed">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_fan_speed_slider_indicator">
     <property name="text">
      <string>0%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="pushButton_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButton_create_new_profile">
     <property name="text">
      <string>Create new</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
